draft-ietf-quic-transport.md

```diff
@@ -2461,9 +2466,10 @@ Any frame type that mentions a stream ID can be sent in this state.
 ### half-closed (local)
 
 A stream that is in the "half-closed (local)" state MUST NOT be used for sending
-on new STREAM frames.  Retransmission of data that has already been sent on
+new STREAM frames.  Retransmission of data that has already been sent on
 STREAM frames is permitted.  An endpoint MAY also send MAX_STREAM_DATA and
-RST_STREAM in this state.
+RST_STREAM in this state.  A stream immediately transitions to
+"half-closed (local)" if it is created with the UNI flag set.
```

Comment:
There's something about this that seems odd. An endpoint opens a stream with UNI, transitions to half-closed (local) and the MUST NOT send new STREAM frames?